.gitignore

```diff
@@ -0,0 +1 @@
+/vendor/
```

Comment:
Please just write `vendor` and perform a `git rm --cached -r vendor` followed by `git commit -m 'Removed vendor directory' && git push`.

Reply:
solved